vagrant/provisioning/roles/haproxy/tasks/main.yml

```diff
@@ -226,14 +229,217 @@
         server foia0 {{ internal_host }}:488 check fall 3 rise 2 check ssl verify required ca-file {{ ssl_ca }}
   when: foia_portal_context is defined
 
-- name: haprxy logrotate
+- name: Set up proxies to Confluent Platform services
+  block:
+    - name: Populate haproxy.conf with fronted part 
+      become: yes
+      marker: "# {mark} ANSIBLE MANAGED BLOCK CONFLUENT PLATFORM CONTEXT"
+      insertafter: "#arkcase.next services"
+      path: "/etc/opt/rh/rh-haproxy18/haproxy/haproxy.cfg"
+      block: |
+        acl acl_schema-registry path_beg /schema-registry
+        acl acl_control-center path_beg /control-center
+        acl acl_config path_beg /config
+        acl acl_control-center-ws path_beg /ws
+    
+    - name: Tell fronted part which backend servers will use
+      become: yes
+      marker: "# {mark} ANSIBLE MANAGED BLOCK CONFLUENT PLATFORM CONTEXT"
```

Comment:
Each `marker` has to be unique; since these two markers are identical, Ansible will overwrite the acl section with this use_backend section.  Maybe add `ACL` to the first marker, and `FRONTEND` to the second marker; something like that.

Reply:
Thanks @david-oc-miller , I've just solved it